docs/CIP/CIP-XXXX/src/cardano-service.yml

```diff
@@ -0,0 +1,664 @@
+asyncapi: 3.0.0
+info:
+  title: Wallet API
+  version: 1.0.0
+  description: >-
+    The primary goal of a well-designed API for a multi-chain digital wallet is
+    to provide the data required to construct transactions and to allow deriving
+    the current state of the wallet while the set of subscribed blockchains is
+    continuously extended. This includes aggregating transactions and on-chain
+    events to present users their transaction history, current balance, as well
+    as chain-specific features such as stake delegation or staking rewards.
+
+
+    The proposed API design offers different endpoints to retrieve the same data
+    in order to support a wide range of edge clients, in particular clients with
+    an intermittent connection or that are bandwidth-constrained.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+defaultContentType: application/json
+servers:
+  cardano-mainnet:
```

Comment:
```suggestion
  127.0.0.1:{port}
```

Reply:
Since this is like a name, I changed it to `local-server` 8dca51e8dd91757734bd6a448fc9afe1aa670e90